ui/plugin-system/src/utils/variables.test.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { parseTemplateVariables, replaceTemplateVariable, replaceTemplateVariables } from './variables';
+
```

Comment:
Is it worth adding a test to cover the use case where someone uses a template variable that isn't defined? I could totally see that accidentally happening with typos or some larger dashboards where a variable gets renamed or removed or something like that.

Reply:
Also, I think this was part of the file moves, so feel free to ignore the comment if you weren't actually modifying this code, unless you're feeling like adding more tests today.